akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteNodeShutdownAndComesBackSpec.scala

```diff
@@ -118,9 +118,8 @@ abstract class RemoteNodeShutdownAndComesBackSpec
         watch(subjectNew)
 
         subjectNew ! "shutdown"
-        fishForMessage(5.seconds) {
-          case _: ActorIdentity       ⇒ false
-          case Terminated(subjectNew) ⇒ true
+        receiveWhile(5.seconds) {
+          case _: ActorIdentity ⇒ true
```

Comment:
a comment here would be great, saying that we expect Terminated but alright if it doesn't arrive